.github/workflows/ci.yml

```diff
@@ -96,12 +96,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - uses: harmon758/postgresql-action@v1
+    - uses: CasperWA/postgresql-action@v1.2
       with:
-        postgresql version: '11'
+        postgresql version: '10'
```

Comment:
Can I quickly ask why the downgrade?

Reply:
To match the installed postgres version in the system otherwise (latest installed by default for ubuntu-18.04 is v10). Oh, and I forgot to write that we need to install postgres still (note that I have also specified the version there), since otherwise we cannot use `PGTest`.

Reply:
However, I have (successfully) removed the explicit installation of postgres-server tools and postgres-client (but I think that one is still installed as a dependency of postgres) ... But yeah, that's life.